example-monorepos/blank/apps/expo/metro.config.js

```diff
@@ -16,10 +16,4 @@ config.resolver.nodeModulesPath = [
   path.resolve(workspaceRoot, 'node_modules'),
```

Comment:
At line 14 solito uses `config.resolver.nodeModulesPath` and on Expo docs and Metro docs tells us to use `config.resolver.nodeModulesPaths`.

Metro Docs: https://facebook.github.io/metro/docs/configuration/#nodemodulespaths
Expo Docs: https://docs.expo.dev/guides/monorepos/#modify-the-metro-config

What if we change it to `nodeModulesPaths` too

Reply:
we can pull this into the other PR if needed. I believe this is copying their monorepo metro example.